docs/pages/2 - Configuring Mill.md

```diff
@@ -182,21 +182,46 @@ configuration options apply.
 
 ## Custom Test Frameworks
 
+Integrating with test frameworks like Scalatest or specs2 is simply a matter of adding it
+to `ivyDeps` and specifying the `testFrameworks` you want to use.
+
+Scalatest example:
+
+
 ```scala
 // build.sc
 import mill._, scalalib._
 
 object foo extends ScalaModule {
   def scalaVersion = "2.12.4"
-  def ivyDeps = Agg(ivy"org.scalatest::scalatest:3.0.4")
-  def testFrameworks = Seq("org.scalatest.tools.Framework")
+
+  object test extends Tests {
+    def ivyDeps = Agg(ivy"org.scalatest::scalatest:3.0.4")
+    def testFrameworks = Seq("org.scalatest.tools.Framework")
+  }
 }
 ```
 
-Integrating with test frameworks like Scalatest is simply a matter of adding it
-to `ivyDeps` and specifying the `testFrameworks` you want to use. After that you
-can [add a test suite](#adding-a-test-suite) and `mill foo.test` as usual,
-passing args to the test suite via `mill foo.test arg1 arg2 arg3`
+Specs2 example:
+
+```scala
+// build.sc
+import mill._, scalalib._
+
+object foo extends ScalaModule {
+  def scalaVersion = "2.12.4"
+
+  object test extends Tests {
+    def ivyDeps = Agg(ivy"org.specs2::specs2-core:4.6.0")
+    def testFrameworks = Seq("org.specs2.runner.Specs2Framework")
+    def scalacOptions = foo.scalacOptions() ++ Seq(
+      "-Yrangepos"
+    )
```

Comment:
Why is that? Is that needed to run specs2? If so, could you please add a comment to the example?

Reply:
Yes, needed by specs2: https://etorreborre.github.io/specs2/website/SPECS2-4.6.0/quickstart.html

Comment added in code to clarify.